### DISCUSSION TITLE
How to filter a table with created time?

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#5958] [Help]
I am trying to filter a table with data that is relevant to today and yesterday dates, how to do this dynamic

**Comment by melohagan** [comment]
Hey @newtonlaw95 

If you want to apply a filter to get the data from the past two days, you can do it with this filter:
![Screenshot 2022-05-18 at 13 50 24](https://user-images.githubusercontent.com/101575380/169042540-f2488c39-af94-4f6c-ac05-a2890ec67fa4.png)

JavaScript:
```
let yesterday = new Date();
yesterday.setDate(yesterday.getDate()-1);

return yesterday.toDateString();
```

**Reply by ghost** [reply]
@melohagan Tried the way you said and its returning data that is related to past one day excluding today date related data.

**Reply by melohagan** [reply]
Hmm that's odd. Is your 'Date' column definitely a Date/Time type?

**Reply by ghost** [reply]
Yes , The column which i am referring is Date/Time type and the data source is MYSQL DB

**Reply by melohagan** [reply]
Does it work if you remove the 'toDateString()' and do `return yesterday;`

**Reply by ghost** [reply]
Tried removing  'toDateString()' but still the data is returning related to past one day excluding today date related data

**Reply by melohagan** [reply]
Would you be able to share your app export please? I've tried to recreate this with MySQL but I can't.
